specreduce/tests/test_tracing.py

```diff
@@ -151,7 +151,7 @@ def test_fit_trace():
     window = 10
     guess = int(nrows / 2)
     img_win_nans = img.copy()
-    img_win_nans[guess - window : guess + window] = np.nan
+    img_win_nans[guess - window: guess + window] = np.nan
```

Comment:
My local flake8 run was complaining about the whitespace but it is strictly unrelated. I can revert such changes if it bothers you.